main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import "fmt"
+
+func max(words []string) string {
+	maxLength := len(words[0])
+	maxWordId := 0
+	for i, n := range words {
+		if maxLength < len(n) {
```

Comment:
Please use `utf8.RuneCountInString()` function to get the length of string. Example you can find in the first lecture.